plugins/theme_oomox/oomox_plugin.py

```diff
@@ -99,6 +99,21 @@ class Plugin(OomoxThemePlugin):
         'GTK3_GENERATE_DARK',
     ]
 
+    theme_model_gtk = [
```

Comment:
i think for caret will be better to just create its own sub-section named "Text Input Caret" (under section "Theme") after theme options will be splitted by StackSwitcher into main sections (Theme/Icons/Terminal/Other)

as i can judge from the mockups of settings screens and incorporating into textstyle recommendations (to avoid repeating labels and prefer grouping of those items instead) i think splitting theme options into smaller sub-section could make sense

Reply:
Personally I feel pretty strange that some color options are not in the Colors section.

What about moving "Caret Aspect Ratio" to "Advanced Options" as [my latest mockup](https://github.com/themix-project/oomox/issues/190#issuecomment-454480311)?

Reply:
i think rather than having some hardcoded options i think it would make sense to group some specific options, for example gtk3-only or caret-specific

but specifically with caret my main motivation -- i really tired of questions what the "caret" is, and when it's under its own section there it's written "Text Input Caret" it's at least giving a hint what the caret is and allows also to avoid repetitive "prefix" "Text Input Caret ..." in options names

idk, mb try to come up with shorter but cleaner labels to solve the issue i've illustrated above, mb "Text Caret 1", "Text Caret 2" in colors section and "Text Caret Aspect Ratio" in other options

Reply:
and also put caret color options close to textbox color options

Reply:
or even Text Cursor instead of Text Caret

Reply:
As another option: What do you think about removing those "caret" options from Oomox theme? To be honest, I think those "caret" options are pretty niche. Even me, I don't know how meaningful/effective it is to change the "Caret Aspect Ratio". I think majority people don't understand what they are, while only quite a few people may need those options.

Reply:
Or leave only "Primary Caret" as "Text Caret", etc.

Reply:
caret options were added by Hypra.fr developer (they ship oomox in Linux distribution for people with visual impairment problems) and, as i agree with them, it's quite important option for making theme more high-contrast (i also added myself configuration of outline sizes with the same reasoning)

also i myself use in gtk almost "square" caret, like in terminal (it's just much faster to locate it on the screen)

Reply:
I see. So I think "Text Caret", "Text Caret in RTL Scripts" in colors section and "Text Caret Aspect Ratio" (or "Aspect Ratio of Text Caret"?) in other options is the most clear at the moment.

> also put caret color options close to textbox color options

That also would be nice, if possible.

Reply:
i think BiDi is fine, since HiDPI is already used, and it's more clear than "Secondary"

Reply:
`Textbox Caret`, `BiDi Textbox Caret`, and `Textbox Caret Aspect Ratio`?

Hmm, I personally feel `Text Caret` sounds more natural and enough, though I'm not good at English...

Reply:
> why does Bi-Directional caret require separate styling from a primary one?

See https://bugzilla.gnome.org/show_bug.cgi?id=764204

Reply:
everyday you learn something new :-)

i don't have strong preference towards text/textbox but it just seems more inline with 'textbox text' and 'textbox background'

Reply:
or just leave caret related stuff to live in it's own section below other gtk options, as now :)

Reply:
Renamed Caret options.

> or just leave caret related stuff to live in it's own section below other gtk options, as now :)

 One of my motivations of this PR is to make comparisons with other colors easier. In the current GUI, long scrolling is required when you want to compare Caret colors and other colors :)